### ISSUE TITLE
README.md

### ISSUE THREAD

**Issue by heistings** [rs/SDWebImage#1055]
README is not updated for `sd_setImageWithURL:`

```
#import <SDWebImage/UIImageView+WebCache.h>

...objective-c

- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
{
    static NSString *MyIdentifier = @"MyIdentifier";

    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:MyIdentifier];

    if (cell == nil)
    {
        cell = [[[UITableViewCell alloc] initWithStyle:UITableViewCellStyleDefault
                                       reuseIdentifier:MyIdentifier] autorelease];
    }

    // Here we use the new provided setImageWithURL: method to load the web image
    [cell.imageView setImageWithURL:[NSURL URLWithString:@"http://www.domain.com/path/to/image.jpg"]
                   placeholderImage:[UIImage imageNamed:@"placeholder.png"]];

    cell.textLabel.text = @"My Text";
    return cell;
}
```

**Comment by rs** [comment]
Could you send a pull request please?

**Comment by heistings** [comment]
My pleasure:)

**Comment by bpoplauschi** [comment]
Fixed by #1057
